exporter/stackdriver/stats.go

```diff
@@ -237,28 +227,26 @@ func (e *statsExporter) makeReq(vds []*stats.ViewData, limit int) []*monitoringp
 
 // createMeasure creates a MetricDescriptor for the given view data in Stackdriver Monitoring.
 // An error will be returned if there is already a metric descriptor created with the same name
-// but it has a different aggregation, window or keys.
+// but it has a different aggregation or keys.
 func (e *statsExporter) createMeasure(ctx context.Context, vd *stats.ViewData) error {
 	e.createdViewsMu.Lock()
 	defer e.createdViewsMu.Unlock()
 
 	m := vd.View.Measure()
 	agg := vd.View.Aggregation()
-	window := vd.View.Window()
 	tagKeys := vd.View.TagKeys()
 	viewName := vd.View.Name()
 
 	if md, ok := e.createdViews[viewName]; ok {
-		// Check agg, window and keys.
-		return equalAggWindowTagKeys(md, agg, window, tagKeys)
+		return equalAggWindowTagKeys(md, agg, tagKeys)
```

Comment:
We still never changed this helper s/equalAggWindowTagKeys/equalAggTagKeys/g

Reply:
oops, changing